docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -9,6 +9,15 @@ info:
     See [the API overview](../api/) for an introduction to the API and information
     on how authorization works.
 
+    ## Expanding Resources
+    Resources often contain links to related resources in their response bodies.
+    Some endpoints support resource expansion, allowing those related resources
+    to be expanded inline in the response.
+
+    To expand resources (where supported), use the `expand` request parameter.
```

Comment:
It wasn't completely clear to me from reading the docs a) what the format of the `expand` parameter should be (field name?, resource type?) and b) how multiple resources to expand should be specified.